src/components/home/enhanced-traffic-stats.tsx

```diff
@@ -178,19 +178,16 @@ export const EnhancedTrafficStats = () => {
   // 是否显示流量图表
   const trafficGraph = verge?.traffic_graph ?? true;
 
-  // WebSocket引用
-  const socketRefs = useRef<{
-    traffic: ReturnType<typeof createAuthSockette> | null;
-    memory: ReturnType<typeof createAuthSockette> | null;
-  }>({
-    traffic: null,
-    memory: null,
-  });
```

Comment:
There is only one effect that creates sockets. We can store this data directly inside that effect.